test/regress/regress1/sygus/incremental-stream-ex.sy

```diff
@@ -0,0 +1,28 @@
+; COMMAND-LINE: -i --sygus-out=status
+;EXPECT: unsat
+;EXPECT: unsat
+;EXPECT: unsat
+;EXPECT: unsat
```

Comment:
This test does not ensure that the results of `check-synth-next` are different. How about adding this one?
```
; COMMAND-LINE: -i
; SCRUBBER: grep "define-fun" | sort
; EXPECT: (define-fun f ((x Int) (y Int)) Int 0)
; EXPECT: (define-fun f ((x Int) (y Int)) Int 1)
; EXPECT: (define-fun f ((x Int) (y Int)) Int x)
; EXPECT: (define-fun f ((x Int) (y Int)) Int y)

(set-logic LIA)

(synth-fun f ((x Int) (y Int)) Int
  ((Start Int))
  ((Start Int (0 1 x y))))

(check-synth)
(check-synth-next)
(check-synth-next)
(check-synth-next)
```
It requires #7819 though...

Reply:
Sounds good.

Reply:
I'll add this regression after that PR is merged.